app/carbon-fields/cf-theme-options.php

```diff
@@ -95,9 +95,19 @@ function crb_attach_theme_options()
             Field
                 ::make('html', 'crb_alps_version')
                 ->set_html(__('<h3>ALPS CORE Version</h3><p style="font-size:16px">Stick to the selected version of ALPS core CSS and Javascript.</p>', 'alps')),
+            Field
+                ::make('checkbox', 'project_alps_version', __('Use project ALPS CORE Version', 'alps'))
+                ->set_option_value('false')
+                ->set_help_text(__('Select project ALPS core version or choose another version.', 'alps'))
+                ->set_width(33),
```

Comment:
Replace the field above with the following: 
```
            Field
                ::make('radio', 'project_alps_version', __('ALPS Core files version', 'alps'))
                ->add_options([
                    'alps-remote' => __('REMOTE: This option pulls the ALPS CORE CSS and Javascript from the remote CDN.', 'alps'),
                    'alps-local' => __('LOCAL: This option caches the most recent version of ALPS to your local site, reducing the connections to remote servers.', 'alps'),
                ])
                ->set_width(33),
```

Reply:
Fixed